src/components/context-menu/ContextMenu.android.js

```diff
@@ -60,7 +60,9 @@ export default function ContextMenu({
           onPress={handleShowActionSheet}
           radiusAndroid={20}
         >
-          {children || <ContextButton {...omit(props, ActionSheetProps)} />}
+          {children || (
+            <ContextButton {...omitFlatten(props, ActionSheetProps)} />
+          )}
```

Comment:
Oo, so here you change that part from another PR that I commented on 😅.

Reply:
usually, I don't change the old naming, but yea, in this case, it makes sense to update it